modules/appeals_api/app/swagger/appeals_api/v1/notice_of_disagreements_controller_swagger.rb

```diff
@@ -471,7 +471,11 @@ class AppealsApi::V1::NoticeOfDisagreementsControllerSwagger
     operation :post, tags: NOD_TAG do
       key :operationId, 'postNoticeOfDisagreementEvidenceSubmission'
       key :summary, 'Get a location for subsequent evidence submission document upload PUT request'
-      key :description, ''
+      key :description, <<~DESC
+        This is the first step to submitting supporting evidence for an NOD.  (See the Evidence Uploads section above for additional information.)
+
+        The Notice of Disagreement GUID that is returned when the NOD is submitted, is supplied to this endpoint to ensure the NOD is in a valid state for sending supporting evidence documents.  Only NODs that selected the Evidence Submission lane are allowed to submit evidence documents up to 90 days after the NOD is received by VA.
+      DESC
       parameter name: 'uuid', 'in': 'path', required: true, description: 'Associated Notice of Disagreement UUID' do
```

Comment:
In the code we're expecting `params[:nod_id]`. Should this be included here? Forgive me, I still don't completely understand our Swagger documentation :/

Reply:
yeah, it looks like the name of the parameter as documented is `uuid`, which should be changed to `nod_id`. Or perhaps we should change the controller attribute to be more consistent with how we refer to the UUID? 🤔

Reply:
I was thinking that, but I do feel that just calling it `nod_id` is so much more clear than `uuid`, even if it is less consistent.

Reply:
Good points... What about `nod_uuid`?

Reply:
There are points in the evidence submission controller where we are looking for evidence submissions via
`find_by(guid: params[:id])` and times when we are creating new submissions with `AppealsApi::EvidenceSubmissionRequestValidator.new(params[:nod_id]...` For me, the code is much more clear when it identifies that this is the `nod_id`, so for me it's about clarity. If this is only clear for me, then let's change it.